yb-voyager/src/utils/utils.go

```diff
@@ -337,6 +337,37 @@ func ContainsAnySubstringFromSlice(slice []string, s string) bool {
 	return false
 }
 
+func PollForMessageFromOffsetInFile(filePath string, offset int64, message string) error {
+	file, err := os.Open(filePath)
+	if err != nil {
+		return fmt.Errorf("error opening file %s: %v", filePath, err)
+	}
+
+	defer file.Close()
+
+	_, err = file.Seek(offset, 0)
```

Comment:
Since we are anyway reading from the beginning of the file, do we really need offset as an input arg?